modules/raft/src/testFixtures/java/org/apache/ignite/raft/jraft/core/TestCluster.java

```diff
@@ -408,6 +408,26 @@ public void waitLeader() throws InterruptedException {
         }
     }
 
+    /**
+     * Wait until a leader is elected.
+     * @throws InterruptedException
+     * @return Leader
+     */
+    public Node waitAndGetLeader() throws InterruptedException {
+        Node node;
+
+        while (true) {
```

Comment:
Do you sure about need an infinity loop here?

Reply:
changed to an assertion